.github/CONTRIBUTING.md

```diff
@@ -556,6 +556,8 @@ for (var/i in reagents)
 
 * Do not divide when you can easily convert it to multiplication. (ie `4/2` should be done as `4*0.5`)
 
+* The use of `\` to separate single lines into more readable blocks is not banned, however you should use it only where it makes new information more accessible. We do not have a column limit, and mass conversions will not be received well.
```

Comment:
I'm 99% sure backslashes are only necessary for like, macros. I split lines normally a lot without them and they work fine. This should be a generic comment about multilining.

Reply:
~~It's needed for macros and math expressions, which are the main place I've seen them~~
Sure, you're right, this should be more general

Reply:
This is bad (assuming the variables are bigger than just single letters, of course, creating big horizontal drift):
```dm
var/list/codes = list(alpha, bravo, charlie, delta, echo, foxtrot)
```
this is better:
```dm
var/list/codes = list(
	alpha,
	bravo,
	charlie,
	echo,
	foxtrot,
)
```
It's quite clear and reduces chances of merge conflicts.

Reply:
@Rohesie I'm not really sure what you're getting at here, do you want me to insert that as an example?

Reply:
The end parenthese should lower an indentation level, for the record 😄